docs/docker_deployment.md

```diff
@@ -34,13 +34,15 @@ cd opentelemetry-demo/
 
 ## Run Docker Compose
 
-- Start the demo (it can take ~20min the first time the command is executed as
-all the container images will be built):
+- Start the demo:
 
 ```shell
-docker compose up
+docker compose up --no-build
 ```
 
+**Note:** removing the `--no-build` command line option will re-build from main.
```

Comment:
Add a note like this to explain what `--no-build` is

Using `--no-build` will fetch the latest releases docker images from [ghcr](http://ghcr.io/open-telemetry/demo) instead of building them yourself.